doc/DEVELOPER_TOOLING.md

```diff
@@ -116,14 +116,35 @@ here](https://apt.llvm.org/) to your `sources.list`, install the needed packages
 libclang-12-dev llvm-12-dev llvm-12-tools`), and build Cataclysm with CMake,
 adding `-DCATA_CLANG_TIDY_PLUGIN=ON`.
 
+#### Other Linux distributions
+
 On other distributions you will probably need to build `clang-tidy` yourself.
-* Check out the `llvm`, `clang`, and `clang-tools-extra` repositories in the
-  required layout (as described for example
-  [here](https://quuxplusone.github.io/blog/2018/04/16/building-llvm-from-source/).
+* Expect this process to take about 80GB of disk space.
+* Check out the `llvm` project, with e.g.
+  `git clone https://github.com/llvm/llvm-project.git llvm-12`.
+* Enter the newly cloned repo `cd llvm-12`.
+* Switch to the version 12 release `git checkout release/12.x`.
```

Comment:
We could probably replace this with:
```sh
git clone --depth=1 -b release/12.x https://github.com/llvm/llvm-project.git llvm-12
```
unless, that is, we want full copies of all branches

Reply:
Good idea.  Done.